core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -3709,6 +3709,12 @@ void checkCorrelatedMapSubQuery(boolean expand) {
     sql(sql).ok();
   }
 
+  @Test void testNestedStructFieldAccessWhere() {
```

Comment:
The test name does not explicitly say that it must be a single field struct.

Reply:
Renamed to `testNestedStructSingleFieldAccessWhere`